src/frontend/components/SøknadsSteg/EøsSteg/Søker/useEøsForSøker.tsx

```diff
@@ -64,6 +67,24 @@ export const useEøsForSøker = (): {
                 : ok(felt)
     );
 
+    const andreUtbetalinger = useJaNeiSpmFelt({
+        søknadsfelt: søker.andreUtbetalinger,
+        feilmeldingSpråkId: 'eøs-om-deg.utbetalinger.feilmelding',
+    });
+    const {
+        fjernPeriode: fjernAndreUtbetalingsperiode,
+        leggTilPeriode: leggTilAndreUtbetalingsperiode,
+        registrertePerioder: registrerteAndreUtbetalinger,
+    } = usePerioder<IUtbetalingsperiode>(
+        søker.andreUtbetalingsperioder,
+        { andreUtbetalinger },
+        avhengigheter => avhengigheter.andreUtbetalinger.verdi === ESvar.JA,
+        felt =>
+            andreUtbetalinger.verdi === ESvar.JA && felt.verdi.length === 0
+                ? feil(felt, <SpråkTekst id={utbetalingerFeilmelding(false)} />)
```

Comment:
Dette er feilmelding for om man får utbetaling nå inni selve modalen. Her skal det være feilmelding for at man må legge til minst en periode

Reply:
`felles.flereytelser.feilmelding`